packages/flutter_sprites/lib/src/action.dart

```diff
@@ -34,14 +39,18 @@ abstract class Action {
     _finished = false;
   }
 
+  /// The duration of this action in seconds.
   double get duration => 0.0;
 }
 
+/// Callback for setting properties, used by [ActionTween].
 typedef void SetterCallback(dynamic value);
 
 /// The abstract class for an action that changes properties over a time
 /// interval, optionally using an easing curve.
 abstract class ActionInterval extends Action {
+
+  /// Creates a new ActionInterval, call this if you are overriding this class.
```

Comment:
no need to say things like "call this if you are overriding this class" which are true of all constructors.

Useful things to say in constructor definitions are things like what arguments are required, what the valid values are, etc. e.g. can curve be null? can duration be null? can duration be negative?